readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://zvir91.github.io/layout_search-bar-airbnb/)
```

Comment:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/38065883/175106795-c9d08fc4-5c37-4ca6-bf19-28615e090871.png">
Color is wrong here